takeNotes-src/putNotes/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "post",
+  "version": "1.0.0",
+  "description": "POST handler for TakeNotes users API",
```

Comment:
All our handlers do this atm - we will need to make ticket to update these for prod